src/main/java/com/google/sps/servlets/BusinessDataServlet.java

```diff
@@ -94,19 +96,34 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     String addressLatStr = request.getParameter(BUSINESS_ADDRESS_LAT);
     String addressLngStr = request.getParameter(BUSINESS_ADDRESS_LNG);
     float addressLat;
-      if (addressLatStr.isEmpty()) {
-          addressLat = 404;
+    if (addressLatStr.isEmpty()) {
+        addressLat = 404;
```

Comment:
can consider changing 404 to a constant like `NOT_FOUND`?
Actually, I am not sure if 404 is a good implementation? cos for other parameters, frontend will check for `isEmpty()` but for float values, we use 404 for validation?